src/main/java/com/capitalone/dashboard/config/MongoConfig.java

```diff
@@ -58,6 +61,14 @@ public MongoClient mongo() throws Exception {
         MongoClient client;
         LOGGER.info("ReplicaSet" + dbreplicaset);
 
+        if (!StringUtils.isEmpty(dbUrl)) {
```

Comment:
does this take the value from application properties from the implementing component?

Reply:
Yes